source/reference.txt

```diff
@@ -1,9 +1,53 @@
-.. place holder for the reference section TOC
-
 =========
 Reference
 =========
 
+:ref:`c2c-mongosync`
+   The primary process used in {+c2c-product-name+}.
+
+:ref:`c2c-config`
+   Configure :program:`mongosync` instances at startup using a
+   configuration file.
+
+:ref:`c2c-api`
+   The API endpoints to interact with ``mongosync``. 
+
+:ref:`c2c-states`
+   States that ``mongosync`` enters depending on requests received. 
+
+:ref:`c2c-filtered-sync`
+   Inclusion filters to specify which databases and collections to
+   include in sync. 
+
+:ref:`c2c-oplog-sizing`
```

Comment:
[question]

Just double-checking: is this the correct description? I would have expected the ref to be directly related to change streams.